modules/web/src/app/core/components/sidenav/template.html

```diff
@@ -52,53 +92,164 @@
-  </div>
+    <mat-menu #menu="matMenu">
+      <mat-label class="menu-label">etcd Backups</mat-label>
+      <hr />
+      <a mat-menu-item
+         fxLayoutAlign=" center"
+         [routerLink]="getRouterLink(view.Backups)"
+         class="collapsed-sub-menu-item"
+         [ngClass]="getLinkClass(view.Backups)">
+        <span>Automatic Backups</span>
+      </a>
+
+      <a mat-menu-item
+         fxLayoutAlign=" center"
+         [routerLink]="getRouterLink(view.Snapshots)"
+         class="collapsed-sub-menu-item"
+         [ngClass]="getLinkClass(view.Snapshots)">
+        <span>Snapshots</span>
+      </a>
+
+      <a mat-menu-item
+         fxLayoutAlign=" center"
+         [routerLink]="getRouterLink(view.Restores)"
+         class="collapsed-sub-menu-item"
+         [ngClass]="getLinkClass(view.Restores)">
+        <span>Restores</span>
+      </a>
+    </mat-menu>
+  </ng-container>
+
+  <ng-container *ngIf="!isSidenavCollapsed()">
+    <km-side-nav-field label="Access"
+                       icon="km-icon-key">
+      <div [ngClass]="{'active': checkUrl(view.SSHKeys), 'km-disabled': !getMenuItemClass(view.SSHKeys)}"
+           class="sub-menu-item">
+        <a id="km-nav-item-sshkeys"
+           fxLayoutAlign=" center"
+           [routerLink]="getRouterLink(view.SSHKeys)"
+           [ngClass]="getLinkClass(view.SSHKeys)">
+          <span>SSH Keys</span>
+        </a>
+      </div>
+
+      <div [ngClass]="{'active': checkUrl(view.Members), 'km-disabled': !getMenuItemClass(view.Members)}"
+           class="sub-menu-item">
+        <a id="km-nav-item-members"
+           fxLayoutAlign=" center"
+           [routerLink]="getRouterLink(view.Members)"
+           [ngClass]="getLinkClass(view.Members)">
+          <span>Members</span>
+        </a>
+      </div>
+
+      <div [ngClass]="{'active': checkUrl(view.Groups), 'km-disabled': !getMenuItemClass(view.Groups)}"
+           class="sub-menu-item">
+        <a fxLayoutAlign=" center"
+           [routerLink]="getRouterLink(view.Groups)"
+           [ngClass]="getLinkClass(view.Groups)">
+          <span>Groups</span>
+        </a>
+      </div>
+
+      <div [ngClass]="{'active': checkUrl(view.ServiceAccounts), 'km-disabled': !getMenuItemClass(view.ServiceAccounts)}"
+           class="sub-menu-item">
+        <a id="km-nav-item-service-accounts"
+           fxLayoutAlign=" center"
+           [routerLink]="getRouterLink(view.ServiceAccounts)"
+           [ngClass]="getLinkClass(view.ServiceAccounts)">
+          <span>Service Accounts</span></a>
+      </div>
+    </km-side-nav-field>
+  </ng-container>
+
+  <ng-container *ngIf="isSidenavCollapsed()">
+    <button (click)="menuTrigger.openMenu()"
+            class="km-icon-key collapesd-button"
+            [matTooltip]="getTooltip(view.Backups)">
+      <div #menuTrigger="matMenuTrigger"
+           [matMenuTriggerFor]="menu"></div>
+    </button>
+    <mat-menu #menu="matMenu">
```

Comment:
Some class is missing which make expansion of panel very abrupt can you compare with Resources and and etcdBackup matMenuTrigger